docs/director/03-tenancy.md

```diff
@@ -0,0 +1,37 @@
+#Tenancy
```

Comment:
Which token are you talking about?
Token is available in director, not in tenant mapping handler.
 
1. The external tenant is generally in headers. In case of runtime mapping handler, the tenant is in body.headers`.
2. Maybe I don't get the question, but tenant mapping get `tenant` from request (this is external tenant) and map to internal tenant.
3.External tenant is mainly in headers.
4. Could you create the issue?

Ask if something is not clear.

Reply:
I was under the impression that the tenant could also come as part of the Bearer token but it seems that was misunderstanding on my part.

Regarding the IAS integration Im not sure where is best to create the issue as its not part of the compass source.

Reply:
Yes the tenant is part of bearer token. The token is constructed after mapping.
Here is detailed description:
https://github.com/kyma-incubator/compass/blob/abb4213a2316e3d051306fad65d23a7c03c4ca7f/docs/compass/03-01-security.md#oauth-20-access-token